diff --git a/node_modules/cross-fetch/dist/browser-ponyfill.js b/node_modules/cross-fetch/dist/browser-ponyfill.js
index 7a795ac..fd217fb 100644
--- a/node_modules/cross-fetch/dist/browser-ponyfill.js
+++ b/node_modules/cross-fetch/dist/browser-ponyfill.js
@@ -465,10 +465,17 @@ var irrelevant = (function (exports) {
       }
 
       xhr.onload = function() {
+        let responseHeaders = xhr.getAllResponseHeaders();
+        const setCookie = xhr.getResponseHeader('set-cookie');
+        if (setCookie) {
+          responseHeaders = responseHeaders.replace(
+            setCookie, setCookie.replace(/\r?\n/g, ', ')
+          );
+        }
         var options = {
           status: xhr.status,
           statusText: xhr.statusText,
-          headers: parseHeaders(xhr.getAllResponseHeaders() || '')
+          headers: parseHeaders(responseHeaders || '')
         };
         options.url = 'responseURL' in xhr ? xhr.responseURL : options.headers.get('X-Request-URL');
         var body = 'response' in xhr ? xhr.response : xhr.responseText;
