diff --git a/node_modules/wikibase-edit/lib/request/fetch.js b/node_modules/wikibase-edit/lib/request/fetch.js
index 127cadb..faebc22 100644
--- a/node_modules/wikibase-edit/lib/request/fetch.js
+++ b/node_modules/wikibase-edit/lib/request/fetch.js
@@ -13,7 +13,7 @@ export async function customFetch (url, { timeout, ...options } = {}) {
 
 // Based on https://stackoverflow.com/questions/46946380/fetch-api-request-timeout#57888548
 function fetchWithTimeout (url, options, timeoutMs = 120_000) {
-  const controller = new AbortController()
+  const controller = new window.AbortController()
   const promise = fetch(url, {
     keepalive: true,
     signal: controller.signal,
diff --git a/node_modules/wikibase-edit/lib/request/login.js b/node_modules/wikibase-edit/lib/request/login.js
index ab914e1..e4cd400 100644
--- a/node_modules/wikibase-edit/lib/request/login.js
+++ b/node_modules/wikibase-edit/lib/request/login.js
@@ -44,6 +44,11 @@ const getSessionCookies = async (loginUrl, config, headers, loginCookies) => {
 
   const resBody = await parseResponseBody(res)
   if (resBody.login.result !== 'Success') {
+    if (
+      resBody.login.reason == 'Unable to continue login. Your session most likely timed out.'
+    ) {
+      throw error_.new('failed to login: session timed out or no login cookies provided')
+    }
     throw error_.new('failed to login: invalid username/password')
   }
 
