diff --git a/node_modules/wikibase-edit/lib/properties/fetch_properties_datatypes.js b/node_modules/wikibase-edit/lib/properties/fetch_properties_datatypes.js
index 6693bca..b059f25 100644
--- a/node_modules/wikibase-edit/lib/properties/fetch_properties_datatypes.js
+++ b/node_modules/wikibase-edit/lib/properties/fetch_properties_datatypes.js
@@ -23,7 +23,8 @@ module.exports = async (config, propertyIds = []) => {
 
   const urls = WBK(instance).getManyEntities({ ids: missingPropertyIds, props: 'info' })
 
-  const responses = await Promise.all(urls.map(getJson))
+  const headers = { 'user-agent': config.userAgent }
+  const responses = await Promise.all(urls.map((url) => getJson(url, { headers })))
   const responsesEntities = responses.map(parseResponse)
   const allEntities = Object.assign(...responsesEntities)
   missingPropertyIds.forEach(addMissingProperty(allEntities, properties))
diff --git a/node_modules/wikibase-edit/lib/request/post.js b/node_modules/wikibase-edit/lib/request/post.js
index 0af9984..915cfb4 100644
--- a/node_modules/wikibase-edit/lib/request/post.js
+++ b/node_modules/wikibase-edit/lib/request/post.js
@@ -57,7 +57,7 @@ const actionPost = (action, data, config) => authData => {
   if (summary != null) data.summary = data.summary || summary
   if (baserevid != null) data.baserevid = data.baserevid || baserevid
   if (tags != null) data.tags = data.tags || tags.join('|')
-  data.maxlag = maxlag || 5
+  if (maxlag != null) data.maxlag = maxlag || 5
 
   params.body = data
 
