diff --git a/node_modules/wikibase-edit/lib/request/post.js b/node_modules/wikibase-edit/lib/request/post.js
index 9c96f76..db82774 100644
--- a/node_modules/wikibase-edit/lib/request/post.js
+++ b/node_modules/wikibase-edit/lib/request/post.js
@@ -67,7 +67,7 @@ const actionPost = (action, data, config) => authData => {
 
   return request('post', params)
   .then(throwErrorRes(`action=${action} error`, params))
-  .then(afterRequest)
+  .finally(afterRequest)
 }
 
 const mayBeSolvedByTokenRefresh = err => {
