diff --git a/node_modules/zotero-types/types/xpcom/db.d.ts b/node_modules/zotero-types/types/xpcom/db.d.ts
index 8a2aeb1..8f9bf2d 100644
--- a/node_modules/zotero-types/types/xpcom/db.d.ts
+++ b/node_modules/zotero-types/types/xpcom/db.d.ts
@@ -70,12 +70,12 @@ declare namespace _ZoteroTypes {
     executeTransaction<T>(
       func: () => T | Promise<T>,
       options?: {
-        disbledForeignKeys: boolean;
-        timeout: number;
-        vacuumOnCommit: boolean;
-        inBackup: boolean;
-        onCommit: (id: string) => void;
-        onRollback: (id: string) => void;
+        disbledForeignKeys?: boolean;
+        timeout?: number;
+        vacuumOnCommit?: boolean;
+        inBackup?: boolean;
+        onCommit?: (id: string) => void;
+        onRollback?: (id: string) => void;
       },
     ): Promise<T>;
 
