diff --git a/node_modules/zotero-types/types/xpcom/db.d.ts b/node_modules/zotero-types/types/xpcom/db.d.ts
index 78dd71d..0f1f9d2 100644
--- a/node_modules/zotero-types/types/xpcom/db.d.ts
+++ b/node_modules/zotero-types/types/xpcom/db.d.ts
@@ -56,7 +56,7 @@ declare namespace _ZoteroTypes {
      *    transaction and re-enable after. (`PRAGMA foreign_keys=0|1` is a no-op during a transaction.)
      * @return {Promise} - Promise for result of generator function
      */
-    executeTransaction<T>(func: () => T | Promise<T>, options?: {disbledForeignKeys: boolean, timeout: number, vacuumOnCommit: boolean, inBackup: boolean, onCommit: (id: string) => void, onRollback: (id: string) => void}): Promise<T>;
+    executeTransaction<T>(func: () => T | Promise<T>, options?: {disableForeignKeys?: boolean, waitTimeout?: number, vacuumOnCommit?: boolean, inBackup?: boolean, onCommit?: (id: string) => void, onRollback?: (id: string) => void}): Promise<T>;
 
     inTransaction(): boolean;
 
